hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -142,6 +142,15 @@
                  "+sw_images={eval_cmd} echo {sw_images} | sed -E 's/\\s+/,/g'"]
       en_run_modes: ["gen_otp_images_mode"]
     }
+    {
+      name: sw_rom_ext_test_mode
+      sw_images: ["sw/device/silicon_creator/mask_rom/mask_rom:0"]
+      run_opts: ["+sw_build_bin_dir={sw_build_dir}/build-bin",
+                 "+sw_build_device={sw_build_device}",
+                 // Separate individual SW images supplied to +sw_images plusarg by comma.
+                 "+sw_images={eval_cmd} echo {sw_images} | sed -E 's/\\s+/,/g'"]
+      en_run_modes: ["gen_otp_images_mode"]
+    }
```

Comment:
Could be refactored this way:
```
    {
      name: sw_test_mode_common
      run_opts: ["+sw_build_bin_dir={sw_build_dir}/build-bin",
                 "+sw_build_device={sw_build_device}",
                 // Separate individual SW images supplied to +sw_images plusarg by comma.
                 "+sw_images={eval_cmd} echo {sw_images} | sed -E 's/\\s+/,/g'"]
      en_run_modes: ["gen_otp_images_mode"]
    }
    {
      name: sw_test_mode_test_rom
      en_run_modes: ["sw_test_mode_common"]
    }
    {
      name: sw_test_mode_mask_rom
      en_run_modes: ["sw_test_mode_common"]
    }
```

You will need to replace `sw_test_mode` on existing tests with `sw_test_mode_test_rom`, which should be a simple search and replace. 

You could mode the `sw_test_mode_mask_rom` to `chip_mask_rom_tests.hjson` if that needs to capture all mask rom related tests.

Reply:
Fixed.